pkg/storage/db.go

```diff
@@ -60,6 +64,7 @@ type DB struct {
 	sqldb   *sql.DB // this is a unit test seam
 	scripts func(name string) (string, error)
 	once    sync.Once
+	now     func() time.Time // unit test seam
```

Comment:
nit: I don't think we need to call these out since this pattern of dependency management is largely inspired by our testing practices.